dap_prinz_green_jobs/notebooks/Similar_occupations.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
great! the most dramatic thing but all the other file names do not have any capital letters so could be saved 'similar_occupations.ipynb`

I took a look at the graphs (love it!) and a few comments:
- i would remove the occupation y axis title as the occupation names often are too long and go over it
- i would use the clean occupation name first to get rid of n.e.c - the function exists in process_ojo_green_measures.py as `clean_soc_name`
- i would rename the occ_greenness legend to "occupation greenness"
- there are a few occupations that only have 1 similar occupation - do we want to filter out those occupations? i don't know what the right answer is because its still interesting so up to you really